alumnos/aoc/dia2/ejercicio4.js

```diff
@@ -0,0 +1,10 @@
+const maxDivider = (divider, limit) => {
+  for(let i = divider; i > 0; i--) {
+    if(divider % i === 0 && i < limit) {
```

Comment:
Creo que aquí has mezclado cosas, me explico, el límite sería el que debería marcar tu for, no el divisor 😅 Además, el divisor siempre debería ser será menor que límite, por lo que la segunda condición siempre se va a cumplir 😅